.meepctl-repocfg.yaml

```diff
@@ -180,16 +180,17 @@ repo:
       src: go-packages/meep-watchdog
       lint: true
   dep:
-    docker-registry:
-      chart: charts/docker-registry
     couchdb:
       chart: charts/couchdb
+    docker-registry:
+      chart: charts/docker-registry
+    grafana:
+      chart: charts/grafana
     influxdb:
       chart: charts/influxdb
+    kube-state-metrics:
```

Comment:
Why did we add kube-state-metrics again?
An ingres controller dependency?

Reply:
I did not add kube-state-metrics here. I just moved it and removed the k8s tag.
Although kube-state-metrics is not currently used, it is available for Grafana dashboards to get metrics from the Kubernetes API.